packages/console/package.json

```diff
@@ -3,9 +3,14 @@
   "version": "0.16.0",
   "description": "Theia - Console Extension",
   "dependencies": {
+    "@phosphor/domutils": "1",
```

Comment:
Why do we do exact match and don't use caret ranges as we do it usually?

Reply:
PhosphorJS is broken up in several bits, and we used to only import one bit and implicitly import the others. If we want to make everything explicit, I was looking for an easy way of declaring it. Just putting `1` means "any version with the major number being 1". This always targets the latest version for major 1, so it is equivalent to `1.0.0`. I don't see the use of being more precise about it, like `^1.2.3`, since we mostly want the latest anyway.

Reply:
If PhosphorJS releases a version 2.0.0, Theia will most likely not be compatible without us updating the API used. This to me essentially mean that we are compatible with v1 in a general manner, hence why it is easier to just say `1`.

Reply:
There won't Phosphor.JS 2. It was deprecated and we going to copy relevant bits in `@theia/core` eventually and inverisfy them.